include/coap3/coap_mutex_internal.h

```diff
@@ -61,16 +64,28 @@ typedef int coap_mutex_t;
 #elif defined(WITH_CONTIKI)
 /* Contiki does not have a mutex API, used as single thread */
 typedef int coap_mutex_t;
-#define COAP_MUTEX_INITIALIZER 0
+#define COAP_MUTEX_DEFINE(_name)			\
+	static coap_mutex_t _name
```

Comment:
So that it is properly initialized, I think I would like to see this as
```
    static coap_mutex_t _name = 0
````
even though it does not really make any difference.

Reply:
Agreed, this initialization should not be done.